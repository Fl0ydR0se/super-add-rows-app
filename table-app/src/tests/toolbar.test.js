import React from "react";

import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { Header } from "../components/ToolBar/Toolbar";

configure({ adapter: new Adapter() });

describe("Toolbar", () => {
  test("should match snapshot ", () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
