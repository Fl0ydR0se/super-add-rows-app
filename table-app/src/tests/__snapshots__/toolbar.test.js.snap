// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar should match snapshot  1`] = `ShallowWrapper {}`;
